Mixpanel/Mixpanel.swift

```diff
@@ -126,6 +132,9 @@ class MixpanelManager {
     }
 
     func removeInstance(name instanceName: String) {
+        if instances[instanceName] === mainInstance {
```

Comment:
do we have something similar to `mainInstance` in Objective-c/Android lib? What is it for?